src/Http/Controllers/MessageController.php

```diff
@@ -146,16 +141,14 @@ public function store(Request $request)
 
       if ($message->wasRecentlyCreated) {
         if ($token) {
-          $meta = ['name' => 'token', 'value' => $token];
-          $message->addMeta($meta, $meta);
+          $message->update(['metas->token' => $token]);
```

Comment:
we can do better than this.

we can create metas->token when message model is being created.
running an update will result to an unnecessary update query.

we could do all at once during creation.